otter/controller.py

```diff
@@ -124,7 +124,19 @@ def pause_scaling_group(log, transaction_id, scaling_group, dispatcher):
                    conv_pause_group_eff(scaling_group, transaction_id))
 
 
-def resume_scaling_group(log, transaction_id, scaling_group):
+def conv_resume_group_eff(trans_id, group):
+    """
+    Resume scaling group of convergence enabled tenant
+    """
+    eff = parallel([
+        Effect(ModifyGroupStatePaused(group, False)),
+        mark_divergent(group.tenant_id, group.uuid).on(
```

Comment:
Totally non-blocking nitpick - since it seems like we want to log "mark-dirty-success" every time we mark a group as divergent, I'm wondering if the logging should be done as part of `mark_divergent` so that functions that use it don't have to look up the exact log event name.